queries.sql

```diff
@@ -0,0 +1,49 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+ROLLBACK;
```

Comment:
- [ ] This code from line 14 to 16 should be inside a separate transaction and there should be a query to verify that the change was made **_before_** and **_after_** the commit. If you take a look at your [second screenshot](https://user-images.githubusercontent.com/44959066/231163127-68ff8a1a-e99e-4021-b16b-aeb3fb65cf91.png) - there is a warning after the commit that reads - **_`there no transaction in progress`_** and this simply means that the command above is not done within a transaction as required. 
- [ ] The ROLLBACK command on line 17 is not necessary. please remove it.

Kindly fix this